run-build.sh

```diff
@@ -163,14 +163,6 @@ if [ $EXIT_CODE != 0 ]; then
     exit $EXIT_CODE
 fi
 
-# Install a project.json based CLI for use by tests
-(set -x ; "$REPOROOT/scripts/obtain/dotnet-install.sh" --channel "master" --install-dir "$DOTNET_INSTALL_DIR_PJ" --architecture "$ARCHITECTURE" --version "1.0.0-preview2-1-003177")
-EXIT_CODE=$?
-if [ $EXIT_CODE != 0 ]; then
-    echo "run-build: Error: installing project-json based cli failed with exit code $EXIT_CODE." >&2
-    exit $EXIT_CODE
-fi
-
```

Comment:
If we are removing the installation of CLI 1.0.0-preview2-1-003177 for non-windows - why would we not remove this for Windows as well?

Reply:
Because I didn't even notice it was there... I needed to remove it here because it was breaking the new distros.

Reply:
Good catch and done!